packages/in_app_purchase/in_app_purchase_storekit/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 0.3.3
```

Comment:
Since this is just a change in the example app, you don't need to bump the version.

Reply:
I was told the exact opposite [here](https://github.com/flutter/plugins/pull/6377#issuecomment-1243927708)😅

Reply:
Apologies. Should it be a patch version then? 0.3.2+1?

Reply:
I've typically avoided it as Dart [recommends](https://dart.dev/tools/pub/versioning#:~:text=avoid%20using%20%2B%20after%20the%20version%20reaches%201.0.0.) not using + after 1.0.0.

Reply:
0.3 is less than 1.0.0 🙂  @jmagman is correct that this should be 0.3.2+1, because we follow the guidance that you linked to:

> Although semantic versioning doesn’t promise any compatibility between versions prior to 1.0.0, the Dart community convention is to treat those versions semantically as well. The interpretation of each number is just shifted down one slot: going from 0.1.2 to 0.2.0 indicates a breaking change, going to 0.1.3 indicates a new feature, and going to 0.1.2+1 indicates a change that doesn’t affect the public API.

Reply:
Silly me, fixed!